rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -26,63 +26,72 @@ Create and document the environment for the automation of processes.
 |*Application Create*, *Application Edit*, *Application Delete*
 
 |===
+* If you have access to an Amazon Web Services (AWS) account and want to create AWS credentials for xref:rpa-builder::toolbox-aws-document-processing.adoc[RPA Builder - Document Processing with AWS], follow the instructions in https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS configuration basics^].
+* If you have access to a Microsoft Azure account and want to create OAuth credentials for xref:rpa-builder::toolbox-mail-operations-mail-session-outlook-with-oauth.adoc[RPA Builder - Mail Session (Outlook with OAuth)]
+, create an application for Outlook, https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/how-to-authenticate-an-ews-application-by-using-oauth[authenticate an EWS application by using OAuth^] and https://learn.microsoft.com/en-us/exchange/client-developer/legacy-protocols/how-to-authenticate-an-imap-pop-smtp-application-by-using-oauth[authenticate an IMAP, POP or SMTP connection using OAuth^]
 +
-Learn how to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[Assign Privileges to a User].
-* To use AWS credentials, you need an AWS account.
-+
-Learn about https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS Configuration basics^]
-* Global variables and credentials can be linked to activity parameters defined in automations. The activity parameters for a process are defined in RPA Builder. Linking global variables or credentials overwrites the initial value of the linked activity parameters in a run configuration.
+If you want to use RPA Builder to xref:rpa-builder::toolbox-variable-handling-credentials-for-oauth.adoc#generating-an-oauth-refresh-token[generate a refresh token for your OAuth credentials], ask an administrator to xref:usermanagement-manage.adoc#assign-users-rpa-builder-access[assign you RPA Builder access].
 
 == Prepare Projects
 
-Projects are assigned to categories. A category describes the environment of a process in a few keywords.
+Project managers organize projects into categories. A category describes the environment of a process in a few keywords.
 
-Applications that are used in processes are assigned to the process and to bots which can use them. A process can be deployed on a bot which has assigned the required processes.
+Administrators list available applications and assign them to bots which can access them. Citizen technologists assign used applications to processes. You can deploy a process on a bot which has assigned the required processes.
```

Comment:
```suggestion
Administrators list available applications and assign them to bots that can access them. Citizen technologists assign used applications to processes. You can deploy a process on a bot which has assigned the required processes.
```

Reply:
Can you say "existing applications" instead of "used applications"? o me, I think of things like "used cars" when I see the term "used."

Reply:
required applications (The process involves using these applications.)